src/app/main.cpp

```diff
@@ -86,6 +86,11 @@ int main( int argc, char **argv )
     return 0;
   }
 
+#if QT_VERSION >= QT_VERSION_CHECK( 6, 5, 1 )
+  // Avoids systematic crash after first launch, a regression in Qt >= 6.5.1
```

Comment:
Can we have a link to an issue or elsewhere, to check if this is still valid in the future?